FiveCalls/FiveCalls/PopoverTip.swift

```diff
@@ -21,18 +21,26 @@ struct PopoverTip: Tip {
 }
 
 extension View {
-    func popoverTipIfApplicable(title: Text, message: Text?) -> some View {
-    if #available(iOS 17, *) {
-        return self
-            .popoverTip(
-                PopoverTip(
-                    title: title,
-                    message: message
-                ),
-                arrowEdge: .top
-            )
-    } else {
-      return self
+    func popoverTipIfApplicable(showingWelcomeScreen: Bool,
+                                title: Text,
+                                message: Text?)
+        -> some View
+    {
+        if #available(iOS 17, *) {
+            if showingWelcomeScreen {
+                AnyView(self)
```

Comment:
I spent a good hour trying to debug this method. If I kept the `return`'s, it compiled, but crashed without a helpful debug message which took me a while to track down. Adding `AnyView` seems to be required once I introduced the additional conditional 🤷🏻‍♂️

Reply:
I ran into this before but didn't figure out how to get it working, `AnyView` to the rescue!